docs/src/main/asciidoc/datasource.adoc

```diff
@@ -178,48 +180,44 @@ Quarkus provides driver extensions for:
 * MySQL - `jdbc-mysql`
 * Oracle - `jdbc-oracle`
 * PostgreSQL - `jdbc-postgresql`
-
-Some other JDBC extensions can be found in the Quarkiverse such as:
-
-* SQLite: https://github.com/quarkiverse/quarkus-jdbc-sqlite[Project homepage] - https://quarkiverse.github.io/quarkiverse-docs/quarkus-jdbc-sqlite/dev/index.html[Documentation]
-
++
```

Comment:
Maybe, but I think you forgot a word or two? It would be:

> To use a JDBC driver for another database, **see the section** 'Use a database with no built-in extension or with a different driver'.

Or just `refer to <<some_link,this section>>` if you think integrating the section title is confusing. I don't know.

Also, the NOTE adminition is not aligned properly, it's nested in the last item even though it's not related to that item.

Reply:
Will patch. Thx for the suggestion.